catalyst-gateway/bin/src/service/api/documents/mod.rs

```diff
@@ -71,17 +76,17 @@ impl DocumentApi {
     )]
     async fn put_document(
         &self, /// The document to PUT
-        document: Cbor<Body>,
+        cbor: Cbor,
```

Comment:
Don't rename this variable, its not `cbor` its a `document`.
Please revert the name